src/js/Ls.js

```diff
@@ -0,0 +1,30 @@
+import * as globals from './globals.js';
+
+class Ls {
+  addToLS = (obj) => {
```

Comment:
Kindly make your class name descriptive that makes your code clear

Reply:
@yohannesdagnachew Changes has been made :)